### ISSUE TITLE
Background permission getting added in the merged manifest

### ISSUE THREAD

**Issue by rananhffc** [BirjuVachhani/locus-android#53]
**Describe the bug**
I loved locus, the implementation was really easy and it handles everything but the only issue is that I do not need background location updates, i only need it on demand, but after implementing locus Background permission is getting added to the merged manifest as well and so i have an issue in uploading it on the play store.

**To Reproduce**
1. Import locus
2. Check merged manifest
**Expected behavior**

As mentioned i require the location only on demand, so is it possible to get only that and not have to take the background location permission?

Thanks in advance!
<img width="716" alt="Screenshot 2021-09-27 at 1 24 24 PM" src="https://user-images.githubusercontent.com/50479641/134867094-1af53e34-8925-437a-8d53-1f5c5d1d7205.png">
ce

**Comment by BirjuVachhani** [comment]
The permission is added into the manifest file of the library. Perhaps I need to make it explicit so the user if wants to use background location. needs to add the permission in the manifest file manually!

**Comment by rananhffc** [comment]
yes that would be helpful and would solve the issue, do close this once you have made the change so that i get informed, i will then reimplement it in my app. Thanks for the wonderful work!

**Comment by iChintanSoni** [comment]
As an alternative, you can place below tag in your manifest:

    <uses-permission tools:node="remove" android:name="android.permission.ACCESS_BACKGROUND_LOCATION" />

**Comment by BirjuVachhani** [comment]
Awesome! I didn't know that. Thanks @iChintanSoni. @rananhffc Does this help in your case? If it does, then I will add this as a note in the readme for other fellow developers who may face what you're facing. Let me know if this works for you!

**Comment by rananhffc** [comment]
hi sorry for the delay in reply, i am yet to implement the manifest remove line, will probably do it by end of this month, but im quite confident that will solve the issue, soo will close this issue. Thanks a ton!

**Comment by BirjuVachhani** [comment]
Great. I'll add this to documentation. Let me know if this does not work. Thanks.

**Comment by russellhoff** [comment]
This week, I haven0t been able to publish my app since Google denied it. The reason? The merged AndroidManifest.xml had the background permission. I removed it as you suggested and they still denied it.

**Comment by Afaquejaya** [comment]
> This week, I haven0t been able to publish my app since Google denied it. The reason? The merged AndroidManifest.xml had the background permission. I removed it as you suggested and they still denied it.

Did you add below the line in your manifest?
`<uses-permission tools:node="remove" android:name="android.permission.ACCESS_BACKGROUND_LOCATION" />`
